js_runner.go

```diff
@@ -66,13 +66,33 @@ func (runner *JSRunner) Init(funcSource string) error {
 		return otto.UndefinedValue()
 	})
 
+	runner.js.Set("console", map[string]interface{}{
+		"error": func(s string) { errorMessageCallback(s)},
+		"log": func(s string) { logMessageCallback(s)},
+	})
+
 	if _, err := runner.SetFunction(funcSource); err != nil {
 		return err
 	}
 
 	return nil
 }
 
+var logMessageCallback = defaultLogFunction
+var errorMessageCallback = defaultLogFunction
+
+func (runner *JSRunner) SetErrorCallback(message func(string)){
```

Comment:
We should rename `message` to something more accurate... e.g:

```suggestion
func (runner *JSRunner) SetErrorCallback(logFn func(string)){
```